Doc: added space requirement to "Build Prerequisites" (#33975)
ceph/ceph: README.md

## Build Prerequisites

> A debug build of Ceph with the source repo takes around 40 gigabytes. If you want to build Ceph in a virtual machine (VM) please make sure total disk space on the VM is at least 60 gigabytes.

Reviewer: A debug build for Ceph may take around 40 gigabytes
```suggestion
A debug build of Ceph may take around 40 gigabytes. If you want to build Ceph in a virtual machine (VM) please make sure total disk space on the VM is at least 60 gigabytes.
```

Reviewer: hey @jbwyatt4 looks good! 
can you squash all commits into 1 keeping commit message as https://github.com/ceph/ceph/pull/33975#issue-388455780 just changing `added to "Build Prerequisites"` to `added space requirement to "Build Prerequisites" `

Author: Will do @ideepika.

Reviewer: > hey @jbwyatt4 looks good!
> can you squash all commits into 1 keeping commit message as [#33975 (comment)](https://github.com/ceph/ceph/pull/33975#issue-388455780) just 

> changing `added to "Build Prerequisites"` to `added space requirement to "Build Prerequisites" `
looks good, can you update commit title?
you can do: 
```
git commit --amend
```
to change commit message

Author: Understood. Amended and force pushed.

> Please also be aware that some distributions of Linux, like CentOS, use Linux Volume Manager (LVM) for the default installation. LVM may reserve a large portion of disk space of a typical sized virtual disk for the operating system.

Reviewer 2: i'd avoid putting more detailed notes or instructions into `README.md`. as personally, i think the longer a document is, the less chance people will read through it.

Author: Removed from README.md

Kept in doc/install/build-ceph.rst
